cups/dest.c

```diff
@@ -138,6 +138,18 @@ typedef struct _cups_namedata_s
   cups_dest_t *dest;                    /* Destination */
 } _cups_namedata_t;
 
+typedef struct _cups_time_s
+{
+#if _WIN32
+  DWORD t;
```

Comment:
I'd actually like to do away with this kind of structure here - let's just use a `double` to return "elapsed time" and then we can use simple math operations portably across platforms.

Reply:
Backported cupsGetClock() as _cupsGetClock().